lib/ui/app_bar.dart

```diff
@@ -18,7 +18,8 @@ abstract class CustomAppBar {
         return buildWithTitle(Strings.profitLossTab);
         break;
       case ARTICLES_TAB:
-        return buildWithTitle(Strings.discoverTab);
+        //return buildWithTitle(Strings.discoverTab);
+        return buildForHome(Strings.discoverTab, profileAction(), homeIcon());
```

Comment:
Probably, we wont want to have access to the profile from this screen.
But we can ask.

Reply:
i put it because is in the UX/UI designs